src/test/java/org/casbin/jcasbin/main/ModelUnitTest.java

```diff
@@ -405,4 +419,24 @@ public void testPriorityModelIndeterminate() {
 
         testEnforce(e, "alice", "data1", "read", false);
     }
+
+    @Test
+    @Parameters(method = "getModels")
+    public void testToString(File file) throws IOException {
+
+        LOGGER.info("Testing toString for {}", file.getName());
```

Comment:
Why don't use logPrint?

Reply:
Because it is inefficient. SLF4J log has both (logEnabled) check and string formatting. As well as accepting more than string. You can pass any Object.